book_extractors/karelians/extraction/extractors/war_data_extractor.py

```diff
@@ -30,5 +37,15 @@ def _nullify_relevant_flags_for_females(self, extracted_data, extraction_results
 
         return extracted_data
 
+    def _transfer_lotta_flag_to_spouse_if_male(self, extracted_data, extraction_results):
```

Comment:
Hmm, mietin olisiko se oikeampi tapa tehdä tämä jotenkin siten, että tämä lotta extractori ajetaan täällä vain jos tämä primary person on mies. Sitten vastaavasti Spouse extractoriin lisäisi tämän lottalipun irroituksen mukaan joka ajettaisiin vain jos kyseessä on nainen? 

Vältyttäisiin sitten tällaisilta kummallisilta extraction_resultsin manipuloinnilta. Ideahan on, että jokaisen aliobjektin kamaa muuttaisi ainoastaan sille määritetty pipeline ja extractorit. Nythän tuonne spouseen sitten vain jotenkin ilmestyy jostain uusi flagi.

Tätä tosin kannattaa sitten hieman miettiä, miten nuo sotajutut spouseen lisätään. Voipi nimittäin olla, että vain osa näistä extractoreista toimii siellä järkevästi, joten saattaa olla ettei sinne kannata tätä `WarDataExtractor`:ia tällaisenaan tunkea, vaan joko suoraan tuo lotta extractor tai kenties ennemmin tekisi tästä extractorista jotenkin konffattavan siten, että se ajaa vain tietyt extractorit (tai excludeaa niitä). Vaatinee hieman miettimistä mutta etköhän jotain keksi ja kysy tarvittaessa niin voidaan funtsia tarkemmin.